devutils/update_submodule.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+UNGOOGLED_REPO=$(dirname $(dirname $(readlink -f ${BASH_SOURCE[0]})))/ungoogled-chromium
+
+# Ensure the submodule is initialized.
+git submodule init
+git submodule update
+
+# Pull latest commits and get the tag submodule is on.
+cd $UNGOOGLED_REPO
+CURRENT_TAG=$(git describe)
+git pull -q
```

Comment:
This will fail on a fresh checkout of the repo with `git pull --recurse-submodules ...`, because `HEAD` will be detached.